lib/parser/objects.ts

```diff
@@ -591,40 +598,48 @@ interface IVariableBase extends IReferenceable, IHasLexerToken, IHasType, IHasDe
   lexerToken: OLexerToken;
 }
 
-export class OFileVariable extends ObjectBase implements IVariableBase {
+export class OFileVariable extends ObjectBase implements IVariableBase, IReferenceable {
+  aliasReferences: OAlias[] = [];
+
   references: OReference[] = [];
   type: ORead[] = [];
   defaultValue?: ORead[] = [];
   lexerToken: OLexerToken;
+  openKind?: ORead[];
+  logicalName?: ORead[];
   constructor(parent: IHasFileVariables, range: OIRange) {
     super((parent as unknown) as ObjectBase, range);
   }
 }
-export class OVariable extends ObjectBase implements IVariableBase {
+export class OVariable extends ObjectBase implements IVariableBase, IReferenceable {
   references: OReference[] = [];
   type: ORead[] = [];
   defaultValue?: ORead[] = [];
   lexerToken: OLexerToken;
+  aliasReferences: OAlias[] = [];
   shared = false;
   constructor(parent: IHasVariables, range: OIRange) {
     super((parent as unknown) as ObjectBase, range);
   }
 }
-export class OSignal extends ObjectBase implements IVariableBase {
+export class OSignal extends ObjectBase implements IVariableBase, IReferenceable {
   references: OReference[] = [];
   type: ORead[] = [];
   defaultValue?: ORead[] = [];
   lexerToken: OLexerToken;
+  aliasReferences: OAlias[] = [];
   registerProcess?: OProcess;
   constructor(parent: (ObjectBase & IHasSignals), range: OIRange) {
     super((parent as unknown) as ObjectBase, range);
   }
 }
-export class OConstant extends ObjectBase implements IVariableBase {
+export class OConstant extends ObjectBase implements IVariableBase, IReferenceable {
```

Comment:
`IVariableBase` already implements `IReferencable`
```suggestion
export class OConstant extends ObjectBase implements IVariableBase {
```